### ISSUE TITLE
Trigger shorthand_operator when using parentheses

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1537]
After #1410, https://github.com/mozilla-mobile/firefox-ios/blob/df12991b0902e6f7b3c7b56d627cee6aee54e7a9/Client/Frontend/Browser/Punycode.swift#L145:17 was fixed, but we should trigger in that case.

```swift
 w = w * (base - t)
```
